src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -2082,6 +2087,12 @@ private val BlockingReason.obstructionFreedomViolationMessage: String get() = wh
     is BlockingReason.Suspended    -> OBSTRUCTION_FREEDOM_SUSPEND_VIOLATION_MESSAGE
 }
 
+enum class ExecutionMode {
```

Comment:
Let's construct the modes with a `String` ID that should be passed to the plugin. Leave a note that these IDs should never be renamed.